students/DrewSmith/session02/series.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+"""
+Module for generating adding sequence index values.
+"""
+def fibonacci(n):
+    """
+    Returns the nth value in the fibonacci sequence.
+    
+    :param n: the zero based index value in fibonacci sequence to return
+    """
+    first , second, val = 0, 1, 0
+    if n <= 0:
+        return first
+    elif n == 1:
+        return second
+
+    for __ in range(2, n + 1):
```

Comment:
The goal was do do it recursively, rather than with a loop.

But you ended up doing that with sum_series later -- so all good :-)